calicoctl/calico_ctl/profile.py

```diff
@@ -35,6 +38,9 @@
       icmp [(type <ICMPTYPE> [(code <ICMPCODE>)])]
            [(from [(tag <SRCTAG>)] [(cidr <SRCCIDR>)])]
            [(to   [(tag <DSTTAG>)] [(cidr <DSTCIDR>)])] |
+      icmpv6 [(type <ICMPTYPE> [(code <ICMPCODE>)])]
+             [(from [(tag <SRCTAG>)] [(cidr <SRCCIDR>)])]
+             [(to   [(tag <DSTTAG>)] [(cidr <DSTCIDR>)])] |
```

Comment:
Shouldn't we be validating the CIDRs also?  ICMPv6 using IPv6 CIDRs and sim. for ICMP (v4).

Maybe worth checking that we don't mix IPv4 and IPv6 source and destination CIDRs too.